src/app/add_entry/page.jsx

```diff
@@ -0,0 +1,157 @@
+"use client"
+import Link from 'next/link'
+import { supabase } from '../lib/supabase'
+import { useRouter } from 'next/navigation'
+import { UploadButton } from '../utils/uploadthing'
+import { useState } from 'react'
```

Comment:
minor suggestion, we tend to split library imports and local file imports. We have a linter to do this. Not at all a big issue issue though, moving on!